utility/src/main/java/org/oppia/android/util/extensions/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+"""
+General purposes utilities to manage extensions
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "bundle_extensions",
+    srcs = [
+        "BundleExtensions.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//third_party:com_google_protobuf_protobuf-lite",
```

Comment:
Why is this needed as a new dependency? It seems like BundleExtensions was building fine before--what changed?

Reply:
@BenHenning the BundleExtensions imports the protobuf package, and the project did not build for me without adding this dependency. We're gonna remove the model library so we needed individual dependencies that just imported
`
import com.google.protobuf.ByteString
import com.google.protobuf.InvalidProtocolBufferException
import com.google.protobuf.MessageLite
`
And this new dependency does that. For reference, look at [this](https://github.com/oppia/oppia-android/issues/2917#issuecomment-803129908) comment.